src/test/ui/borrowck/borrowck-describe-lvalue.mir.stderr

```diff
@@ -367,7 +367,7 @@ error[E0382]: use of moved value: `x`
 LL |                 drop(x);
    |                      - value moved here
 LL |                 drop(x); //[ast]~ ERROR use of moved value: `x`
-   |                      ^ value used here after move
```

Comment:
I think that we'll need to check that `moved_place` doesn't equal `used_place` - since in that case it isn't partial (since it seems like a place is a prefix of itself).

Reply:
When I add this check
`if used_place != moved_place` 
most of tests start failing again. I tried inspecting the `used_place` and `moved_place` values for a single test using `dbg!()`, but I was unable to figure out where to see that output...

Reply:
I've never tried to use `dbg!()` within the compiler, you can use the standard `debug!("{}", x)` logs and then set `RUST_LOG=rustc_mir` when invoking the compiler.

It's possible that the failures you see are just the tests that incorrectly changed previously (ie. the one highlighted in this review comment) and so would change back and fail until blessed.